tests/system/Autoloader/FileLocatorTest.php

```diff
@@ -119,4 +120,100 @@ public function testLocateFileReturnsEmptyWithBadNamespace()
 	}
 
 	//--------------------------------------------------------------------
+        
+        public function testSearchSimple()
+	{
+                $expected = rtrim(APPPATH,'/').'/Views/form.php';
+				
+		$foundFiles = $this->loader->search('Views/form.php');
+                
+		$this->assertEquals($expected,$foundFiles[0]);
+	}
+	
+        //--------------------------------------------------------------------
+        
+        public function testSearchWithFileExtension()
+	{
+                $expected = rtrim(APPPATH,'/').'/Views/form.php';
+				
+		$foundFiles = $this->loader->search('Views/form','php');
```

Comment:
Oops. This one's my fault, but please choose a different file to check against here. That one got added to the repo by accident and needs to be removed. Sorry!